tasks/main.yml

```diff
@@ -4,22 +4,22 @@
 # minio S3 gateway with custom users in front of an external S3 store
 # https://github.com/minio/minio/issues/8045
 
-- name: Install docker-py
+- name: Install docker
   become: true
-  yum:
-    name: docker-python
+  ansible.builtin.pip:
```

Comment:
yum packaging was not following the latest version of the library
installing via pip is different. Not possible to install using dnf on RockyLinux 9?

Reply:
I could not find a dnf installation package for it.

Reply:
this means that we should fix the current version to be sure it does not change every time someone runs the role

Reply:
I have added a new variable for the docker version.